playwright.config.ts

```diff
@@ -0,0 +1,12 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+	webServer: {
+		command: 'npm run build && npm run preview',
+		port: 4173
```

Comment:
Should this be hardcoded?

Reply:
Yes, that's the default and standard port for tests/preview in svelte